purchase_delivery_split_date/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================================================
+Split Purchase Deliveries in one reception per expected date
+============================================================
+
+When this module is installed, each Purchase Order you confirm will
+generate one Incoming Shipment for each expected date indicated in the
```

Comment:
I'd use the word schedule date instead of expected because is what the user sees on the form view.